core/css/decanter.css

```diff
@@ -300,139 +300,181 @@ template {
 
```

Comment:
All the fonts rendered for me in the compiled styleguide with no issues - looks great.

I was scrolling down the diff page on github and saw the file size differences below of fonts removed. For the same font variant, eg:
BIN -91.6 KB core/fonts/crimsontext-bold-webfont.ttf
BIN -41.3 KB core/fonts/crimsontext-bold-webfont.woff

Looks like the woff format in general is about half of that of ttf files. Would that be a reason to keep the woff files instead of the ttf ones?

Reply:
Nice find! This encouraged me to do a little more looking into the file extensions. WOFF is basically OTF or TTF with metadata and compression supported by all major browsers. Because the fonts are compressed, they load faster. This indeed is a reason to use WOFF over TTF. I'll compress the files and upload the new versions.